prov/sm2/src/sm2_msg.c

```diff
@@ -309,17 +309,19 @@ static ssize_t sm2_generic_sendmsg(struct sm2_ep *ep, const struct iovec *iov,
 	total_len = ofi_total_iov_len(iov, iov_count);
 	assert(!(op_flags & FI_INJECT) || total_len <= SM2_INJECT_SIZE);
 
-	ret = sm2_proto_ops[sm2_proto_inject](ep, peer_smr, peer_gid, op, tag,
-					      data, op_flags, FI_HMEM_SYSTEM, 0,
-					      iov, iov_count, total_len);
+	ret = sm2_proto_ops[sm2_proto_inject](
+		ep, peer_smr, peer_gid, op, tag, data, op_flags, FI_HMEM_SYSTEM,
+		0, iov, iov_count, total_len, context);
```

Comment:
lots of changes here for just adding a field...

Reply:
clang-format says sorry... but this is what it likes